addons/mail/static/src/utils/common/format.js

```diff
@@ -6,6 +6,10 @@ import { escape, unaccent } from "@web/core/utils/strings";
 const urlRegexp =
     /\b(?:https?:\/\/\d{1,3}(?:\.\d{1,3}){3}|(?:https?:\/\/|(?:www\.))[-a-z0-9@:%._+~#=\u00C0-\u024F\u1E00-\u1EFF]{2,256}\.[a-z]{2,13})\b(?:[-a-z0-9@:%_+~#?&[\]^|{}`\\'$//=\u00C0-\u024F\u1E00-\u1EFF]|[.]*[-a-z0-9@:%_+~#?&[\]^|{}`\\'$//=\u00C0-\u024F\u1E00-\u1EFF]|,(?!$| )|\.(?!$| |\.)|;(?!$| ))*/gi;
 
+// Match markedown code fences (```xml [...] ```) and inline code (`[...]`),
+// extracted from the marked librairy
```

Comment:
`librairy` => `library`

Also probably best to put in `markdown.js`, so all utils related to marked library are in the same location